src/actions/util.js

```diff
@@ -0,0 +1,50 @@
+import linkifyHtml from "linkifyjs/html";
+import chrono from "chrono-node";
+
+var buildCalendarEvent = (parsedEvent, title) => {
+  let base_url = "https://calendar.google.com/calendar/render?action=TEMPLATE&";
+  let params =
+    "dates=" + parsedEvent.start.moment().format("YYYYMMDD[T]HHmmss[Z]");
+  if (parsedEvent.hasOwnProperty("end")) {
+    params += "/" + parsedEvent.end.moment().format("YYYYMMDD[T]HHmmss[Z]");
+  }
+  params += "&&";
+  params += "text=" + encodeURIComponent(title);
+  return base_url + params;
+};
```

Comment:
Since we're rearranging the code and moving to a more ES6-ish syntax, maybe we can convert these to use template literals as well? Like in [`actions.js`](https://github.com/ghostwriternr/dashboard/pull/8/files#diff-d25573781184593c6cbed0a9c4badccdR27).